.vscode/settings.json

```diff
@@ -3,5 +3,11 @@
 		"*.css": "tailwindcss"
 	},
 	"typescript.tsdk": "./core/node_modules/typescript/lib",
-	"javascript.preferences.autoImportSpecifierExcludeRegexes": ["@prisma\\/client"]
+	"javascript.preferences.autoImportSpecifierExcludeRegexes": ["@prisma\\/client"],
+	// these variables are necessary in order to correctly run seeds
+	"playwright.env": {
+		"NODE_OPTIONS": "--import=\"#register-loader\"",
+		"SKIP_VALIDATION": "true",
+		"DATABASE_URL": "postgresql://postgres:postgres@localhost:54322/postgres"
```

Comment:
the `54322` is the one you get when you do `pnpm dev:setup`, which what i'm assuming you're running if you're running the e2e tests.
(`54323` is the one you get when you run `pnpm test:setup`).

unfortunately these two are atm mutually exclusive, which is very annoying but a separate problem to fix